modules/sdk-coin-zeta/test/unit/zeta.ts

```diff
@@ -336,4 +342,131 @@ describe('Zeta', function () {
       stub.restore();
     });
   });
+
+  describe('Recover transaction: success path', () => {
+    const sandBox = sinon.createSandbox();
+    const destinationAddress = wrwUser.destinationAddress;
+    const coin = coins.get('tzeta');
+    const testBalance = '15000000000000000';
+    const testAccountNumber = '123';
+    const testSequenceNumber = '0';
+    const testChainId = 'test-chain';
+
+    beforeEach(() => {
+      const accountBalance = sandBox.stub(Zeta.prototype, 'getAccountBalance' as keyof Zeta);
+      accountBalance.withArgs(wrwUser.senderAddress).resolves(testBalance);
+
+      const accountDetails = sandBox.stub(Zeta.prototype, 'getAccountDetails' as keyof Zeta);
```

Comment:
can you mock the accountDetails response instead of stubbing it.
Check injective for reference.

Reply:
sure, just to make sure we're calling the expected URL path and don't run into other errors ?

Reply:
yeah correct.